pkg/registry/apis/provisioning/jobs/sync/worker.go

```diff
@@ -93,7 +93,7 @@ func (r *SyncWorker) Process(ctx context.Context, repo repository.Repository, jo
 	}
 
 	syncError := syncJob.run(ctx, *job.Spec.Sync)
-	jobStatus := progress.Complete(ctx, syncJob.run(ctx, *job.Spec.Sync))
+	jobStatus := progress.Complete(ctx, syncError)
```

Comment:
@MissingRoberto this had me very very confused -- we were calling `syncJob.run(` 2x, so the newly added checks to make sure we only have one thing for each identifier was failing 🤣

Reply:
haha, sorry for this! refactoring mistake :)